addons/promote/model-java/src/main/java/org/commonjava/indy/promote/model/GroupPromoteRequest.java

```diff
@@ -93,13 +96,17 @@ public GroupPromoteRequest setDryRun( final boolean dryRun )
     @Override
     public boolean isFireEvents()
     {
-        return false;
+        return fireEvents;
+    }
+
+    public void setFireEvents( boolean fireEvents )
```

Comment:
Hmm, this has me wondering if it's a good idea to allow the user to effectively send in a promotion request in stealth mode...is that a decision they should be allowed to make?